src/game/default/g_types.h

```diff
@@ -515,6 +523,8 @@ typedef struct {
 
 		uint16_t weapon_no_ammo;
 		uint16_t weapon_switch;
+		
+		uint16_t countdown[11];
```

Comment:
`#define NUM_COUNTDOWN_SOUNDS 11`, and then either use `NUM_COUNTDOWN_SOUNDS` or `lengthof(g_media.sounds.countdown)` when iterating, etc.